react/features/filmstrip/components/web/Thumbnail.js

```diff
@@ -1204,8 +1206,9 @@ function _mapStateToProps(state, ownProps): Object {
         _localFlipX: Boolean(localFlipX),
         _participant: participant,
         _raisedHand: hasRaisedHand(participant),
-        _stageFilmstripDisabled: !isStageFilmstripEnabled(state),
-        _stageParticipantsVisible: shouldDisplayStageFilmstrip(state, 1),
+        _stageFilmstripLayout: shouldDisplayStageFilmstrip(state, 0),
```

Comment:
Here we need `shouldDisplayStageFilmstrip` to return `true` even if tile view is displayed. Am I correct?

Reply:
Yes